scripts/verify-spelling-lite.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+# Copyright (C) 2020  Feignint <feignint@gmail.com>
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+set -e
+
+# files to check by default
+# Note: if src/dosbox exists it will also be checked
+check_files=( "*.ac" "*.md" "*.MD" "README*" "dosbox.1" "INSTALL" )
+
+list_files ()
+{
+  [[ ${skip_list_files} ]] ||
+    git -C "${repo_root}" ls-files -- \
+      ':!contrib/translations' \
+      ':!.gitignore' \
+      "${check_files[@]}"
+
+  [[ -e ${repo_root}/src/dosbox ]] &&
+    printf "%s\n" "src/dosbox"
+}
+
+# The following patterns should be ignored
+  filter="README: esc esc"
+filter+="|README: tcl -> Tcl"
+filter+="|stb_vorbis.h: finalY"
+
+# TODO make this:
+filter+="|duplicate word"
+# optional, NOTE: line breaks are ignored so produces many false positives
+# reduce the spam with sed 's/.+/& #/'
+## The problem with that sed is the filename is 'lost', and a loop is slow
+
+check_spelling ()
+{
+  # shellcheck disable=SC2086
+  spellintian $picky "${file_list[@]/#/${repo_root}\/}" #| sed -E '/('"${filter}"')/d'
+}
+
+Get_context ()
+{
+  # get "offending" lines
+  jq -R -s '.[:-1]|split("\n";"")' < <(
+    # shellcheck disable=2046
+    git -C "${repo_root}" grep -n "${TYPO_FIX% ->*}" \
+        $( [[ $( file --mime-type "${repo_root}/${FILENAME%:}" ) =~ app.+exec ]] ||
```

Comment:
This seems to fail on github runner

it works locally

```json
{
  "filename": "src/dosbox",
  "typo": "implemenations",
  "fix": "implementations",
  "context": [
    "src/gui/render_crt_glsl.h:460:\tThese custom implemenations rely on the `rubyTextureSize` uniform variable."
  ]
}
```

Reply:
Is `jq` installed and is it new enough? Maybe some shared MIME database `file` is using is older or not installed?

The default CI VM can be outdated (might need `ubuntu-latest`).

Reply:
yeah, it is probably the MIME db

I specifically targeted the ubuntu-20.04 in the linux.yml

The jq should be fine, it seems to be installed by default in the github runners

jq version could be sensitive, I'll worry about that when doing analysis, at the moment just filtering out "duplicate words"
as they are almost certainly false positives

there  a couple of false positives, which I was filtering ( turned off at the moment )

```
README: tcl -> Tcl
stb_vorbis.h: finalY -> finally 
```

Reply:
```
[[ /home/runner/work/dosbox-staging/dosbox-staging/src/dosbox: application/x-sharedlib =~ app.+exec ]]
```
that is it

I get *application/x-pie-executable*

I guess I should do
```
[[ $foo =~ =~ app.+(x-pie-executable|x-sharedlib) ]]
```